aws_signing_helper/windows_cert_store_signer.go

```diff
@@ -89,6 +92,12 @@ const (
 	WIN_API_FLAG = windows.CRYPT_ACQUIRE_PREFER_NCRYPT_KEY_FLAG
 )
 
+var (
+	// ErrRequiresUI is used when providers are required to display
+	// UI to perform signing operations.
+	ErrRequiresUI = errors.New("provider requries UI to operate")
```

Comment:
Is there a reason to pre-instantiate the error? Seems this happens for `ErrUnsupportedHash` as well.

Reply:
Not really, but maybe it's more readable since you can easily tell what the error is for? It's also not as useful in this case, since the error is used in only one place. Looking at the code base, `ErrUnsupportedHash` is used in three different places.